src/main/java/org/jenkinsci/remoting/engine/EngineUtil.java

```diff
@@ -91,4 +96,24 @@ protected static Properties readResponseHeaders(InputStream inputStream) throws
             response.put(line.substring(0,idx).trim(), line.substring(idx+1).trim());
         }
     }
+
+    /**
+      * Closes the item and logs error to the log in the case of error.
+      * Logging will be performed on the {@code WARNING} level.
+      * @param toClose Item to close. Nothing will happen if it is {@code null}
+      * @param logger Logger, which receives the error
+      * @param closeableName Name of the closeable item
+      * @param closeableOwner String representation of the closeable holder
+      */
+    static void closeAndLogFailures(@CheckForNull Closeable toClose, @Nonnull Logger logger,
```

Comment:
Seems "no", the merge process removed the usage. But it will be definitely used when I start cleaning up FindBugs